docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/:/backend/
+    entrypoint: ./docker-start.sh
+    container_name: taia-x-backend
+    env_file:
+      - ./backend/.env
+    ports:
+      - 5678:5678
+      - 8000:8000
+    depends_on:
+      - db
+      - ipfs
+
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
```

Comment:
@amrap030 I will create here a start up script here instead. The problem with just taking the Dockerfile, is that the depencies (ex: `npm install`) is just run when you build the image (the first time you start the container). If you for example, add new npm packages after the image is built, this packages won't be updated when you restart the container. Because the Dockerfile didn't not change at all and it won't rebuild automatically.